src/spikeinterface/core/node_pipeline.py

```diff
@@ -154,16 +161,26 @@ class SpikeRetriever(PeakSource):
         If False, the max channel is computed for each spike given a radius around the template max channel.
     extremum_channel_inds: dict of int
         The extremum channel index dict given from template.
-    radius_um: float (default 50.)
+    radius_um: float, default: 50.
         The radius to find the real max channel.
         Used only when channel_from_template=False
-    peak_sign: str (default "neg")
+    peak_sign: str, default: "neg"
```

Comment:
```suggestion
    peak_sign: "neg" | "pos" | "both" default: "neg"
```

I would suggest using this notation. Pre-3.10 the preferred notation for this would be `Literal['pos', 'neg']`, because although we input these as strings really there are only a limited number of strings recognized. Another option would be `'pos' or 'neg' or 'both'`, but for users actually using the doc strings it would be clearer to tell them the choice rather than say `str`. I've been slowly trying to update this. But what do you think?